src/EditorFeatures/Core/ExtractMethod/AbstractExtractMethodCommandHandler.cs

```diff
@@ -244,20 +252,5 @@ private bool TryNotifyFailureToUser(Document document, ExtractMethodResult resul
 
             return null;
         }
-
-        /// <summary>
-        /// Applies an ExtractMethodResult to the editor.
-        /// </summary>
-        private void ApplyChangesToBuffer(ExtractMethodResult extractMethodResult, ITextBuffer subjectBuffer, CodeCleanupOptions cleanupOptions, CancellationToken cancellationToken)
-        {
-            using var undoTransaction = _undoManager.GetTextBufferUndoManager(subjectBuffer).TextBufferUndoHistory.CreateTransaction("Extract Method");
-
-            // apply extract method code to buffer
-            var (document, _) = extractMethodResult.GetFormattedDocumentAsync(cleanupOptions, cancellationToken).WaitAndGetResult(cancellationToken);
```

Comment:
the bug was here.  we were throwing away the second part of the result of GetFormattedDocumentASync.  That part of hte result tells us where the name token is to start inline-rename after the formatting occurs.

Reply:
Thanks for the explanation here :)